# Molecular Biology

## DNA extraction protocol
Objectives: to extract DNA from microbial cells of the sample and to sequence them to understand the microbial communities present.
In the laboratory, the main DNA extraction protocol are: 
- PowerSoil kit: it is a quick method, with low content of hazardous chemicals; but there is the risk of contamination and it is expensive;
- Phenol-Chloroform protocol: it is time-consuming and hazardous chemicals are used; it increases chances for contamination and sample mishandling because of the greater hands-on effort and multiple tube transfers involved; but it is cheap.

DNA extraction is divided in different steps:
1) Buffering the extraction;
2) Membrane lysis;
3) Stabilizing DNA;
4) Proteins/Organics removal;
5) DNA separation;
6) DNA recovery.

In the first phases, extraction buffer solution (Tris-HCL, EDTA, NaCl, PBS, CTAB) acts on the membrane layers, buffering the pH, chelating cations used by nucleases (whose action has to be avoided), protecting the negatively charged phosphate groups of the DNA through the positively charge of salts. The consequent cell lysis can be carried out through mechanical effects like (sonication, freezing-thawing) or chemical effects through SDS (sodium dodecyl sulphate) which breaks down the lipids in the cell membrane and the high affinity to bind proteins help to denature them quickly. The enzymatic actions is carried out by: lysozime, that reacts with peptidoglycan layer and breaks the glycosidic bond (ok for Gram+) and it is used in combination with detergents to break the cell wall and  membrane (ok for Gram-); proteinase k, digesting contaminating proteins and protecting the nucleic acids from nuclease attack, by working also in presence in EDTA that chelates cations. 
In the final phases, Phenol:Chloroform:Isoamyl Alcohol (25:24:1) used to promote the partitioning of lipids and cellular debris into the organic phase, leaving isolated DNA in the aqueous phase. In this phase, it is important to consider the salinity of sample because high values can cause variation in densities and to invert the phase in this step of DNA extraction. 
For DNA precipitation, Sodium Acetate is used, by exploiting the positive charges neutralize the negative charge on the phospate groups on the nucleic acids, making the molecule far less hydrophilic and, therefore, much less soluble in water; isopropanol is used because the DNA is less soluble in it; its use at  room temperature is necessary to reduce the salt content that may precipitate with the DNA.
For DNA washing, ethanol is used to remove the excess of salts. The cold alcohol helps the DNA precipitate (solidify and appear) more quickly.
For DNA recovery, Tris buffer is used to solubilize the DNA in an alkaline pH making easier DNA dissolving.

## Electrophoresis gel DNA
Nucleic acid gel electrophoresis is a molecular biology technique that allows for the separation, identification, and purification of DNA and RNA fragments based on size and charge. In this technique, DNA and RNA molecules are separated by applying an electric field that moves the negatively charged nucleic acids through a matrix of:

-   **Agarose**, which is extracted from seaweed, is appropriate for separating DNA fragments ranging in size from a few hundred base pairs to about 20 kb.\
-   **Polyacrylamide** is preferred for proteins and smaller DNA fragments.\
In general, the gel matrix acts like a sieve, allowing shorter molecules to move more quickly through the pores of the gel than longer molecules.\

The equipment and supplies for conducting agarose gel electrophoresis in DNA studies include electrophoresis chamber and power supply, gel casting trays, sample combs, electrophoresis buffer, loading buffer, ethidium bromide (EtBr), and transilluminator.\ 
One method of staining DNA is to expose it to the dye EtBr. EtBr intercalates between the stacked bases of nucleic acids and fluoresces red–orange when illuminated with ultraviolet (UV) light. EtBr should be handled with care for its toxic and mutagenic nature; this is the reason why, it is substituted by other dyes like Sybr Green.\
Agarose gels are cast by melting the agarose powder dissolved in a specific buffer until a clear, transparent solution is achieved. The melted gel is then poured on a tray and allowed to solidify. Upon solidification, the DNA samples are loaded into the sample wells and gel is run at a voltage and for a time period that will perform optimal separation. One well is reserved for a DNA ladder, a standard reference that contains DNA fragments of known lengths. When an electric field is applied across the gel, DNA migrates toward the anode.\
A well-defined “line” of DNA on a gel is called a band. Each band contains a large number of DNA fragments of the same size that have all traveled as a group to the same location. By comparing the bands of a sample to the DNA ladder, we can determine its approximate size.\
For linear fragments of DNA and/or RNA the migration distance is inversely proportional to the size of the molecule (i.e. its length in bases).\
Single-stranded nucleic acids (like RNA) tend to form complex secondary structures, so their "electrophoretic pathway" is strongly influenced by how they fold. Before being separated by electrophoresis, RNA must be denatured to keep the molecules in linear shape. Commonly used denaturing agents: heat, formaldehyde, formamide, urea.\

## PCR
Polymerase chain reaction (PCR) is a technique for exponential ampliﬁcation of a fragment of DNA, using a heat-stable DNA polymerase and an automated heat block that is capable of rapid changes of temperature, inducing the enzymatic synthesis of millions of copies of a specific DNA segment.\
It can be schematized in different steps:

-   **First stage:** The template DNA molecule is initially denatured to two single strands by heating to high temperature (typically 90–95 °C). Two small oligonucleotides that are complementary to sequences on opposite strands of the template molecule are used as primers, allowing the binding for the DNA polymerase.\
-   **Second stage:** the single-stranded oligonucleotide primers anneal to the denatured template molecule, during a cooling phase.\
-   **Third stage:** the new strand extends from the annealed primer in a 5'-3' direction by the heat-stable polymerase. This is performed at the optimum temperature for the polymerase (68–72 °C).\

The most commonly used polymerase is the enzyme isolated from Thermus aquaticus (Taq DNA polymerase).\
After the ﬁrst cycle, each template molecule has been ampliﬁed to two molecules. These in turn are denatured in the next cycle and ampliﬁed to produce four molecules. The four molecules are ampliﬁed to eight in the third cycle, and so on. Each successive cycle effectively doubles the amount of DNA product. The three-stage cycle of denaturation, annealing, and primer extension is repeated 25–40 times in a typical PCR procedure.\
PCR products are visualized by electrophoresis gel technique followed by staining with ﬂuorescent dyes or by hybridization to labeled oligonucleotide probes.\
Species specific primers are used. The most used sequence is rRNA that includes the gene for 16S rRNA.\
The reagents that are used are:

-   **Buffer Tris-HCl:** keeps the pH stable in order to generate an environment suitable for the reaction.\
-   **Primers:** single-stranded oligonucleotides of approximately 18-25 bases that allow the reaction to be started. The primers are complementary, one at the 3' end and the other at the 5' end of the DNA segment you want to amplify. In particular, the forward primer will bind to the 3'-5' antiparallel strand, delimiting the beginning of the synthesis, while the reverse primer will bind to the other strand, delimiting the end of the synthesis.\
-   **$MgCl_{2}$:** Cofactor of Taq DNA polymerase that influences primer hybridization.\
-   **Deoxynucleoside triphosphates (dNTPs-dATP, dTTP, dGTP, dCTP):** they are necessary for the synthesis of new strands.\
-   **Taq DNA polymerase:** heat-stable polymerase.\
-   **DNA sample** of interest.\

## Nanodrop 

## Qubit

## Protein extraction
