# Bioinformatic 

## 16S rRNA analysis
This test analyzes a bacterial gene called 16S rRNA – a sort of identity card for microbes – and provides the name of all the bacteria present in the sample and their abundance as a result.\
It is the most studied gene for microbial phylogeny; it is one of the components of the lower subunit of the bacterial ribosome.\
The gene, that codes for 16S RNA (component of the small subunit of the bacterial ribosome), is approximately 1,500 bp long. A 2-3% change in nucleotide sequence (30-45 bp) is considered indicative of a different species.\
The approach of building phylogenetic trees based on 16S sequence alignment is the most widespread and was used for the first time in 1977 by Carl Woese to distinguish the domain of Archaea from that of Bacteria.\

## Shotgun metagenomic analysis 
High-throughput sequencing approaches enable genomic analyses ideally of all microbes in a sample, not just those that are amenable to cultivation.\
Shotgun metagenomic sequencing allows researchers to comprehensively sample all genes from all organisms in a given complex sample. The method allows microbiologists to evaluate bacterial and metabolic diversity and detect the abundance of microbes in various environments.\
A typical shotgun metagenomics study comprises five steps, after the initial study design:

-   the collection, processing and sequencing of the samples;\
-   preprocessing of the sequencing reads;\
-   sequence analysis to profile taxonomic, functional and genomic features of the microbiome;\
-   statistical and biological post-processing analysis;\
-   validation.\

## Pipelines