<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 5 Molecular Biology | G.Lab. Book</title>
  <meta name="description" content="Book for documentation, tutorial and protocols of the Giovannelli Lab." />
  <meta name="generator" content="bookdown 0.39 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 5 Molecular Biology | G.Lab. Book" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Book for documentation, tutorial and protocols of the Giovannelli Lab." />
  <meta name="github-repo" content="giovannellilab/giovannellilab.github.io" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 5 Molecular Biology | G.Lab. Book" />
  
  <meta name="twitter:description" content="Book for documentation, tutorial and protocols of the Giovannelli Lab." />
  

<meta name="author" content="Gabriella Gallo" />
<meta name="author" content="Davide Corso" />
<meta name="author" content="Donato Giovannelli" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="microbial-metabolisms.html"/>
<link rel="next" href="biogeochemical-cycles.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<div style="text-align:center">
<li class="toc-logo"><a href="./"><img src="img/glab_logo_notext.png" width="200" height="200"></a></li>
<li><a href="./"><strong>G.Lab. Book</strong></a></li>
</div>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#design-principles"><i class="fa fa-check"></i><b>1.1</b> Design principles</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#credits"><i class="fa fa-check"></i><b>1.2</b> Credits</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="basic-knowledge.html"><a href="basic-knowledge.html"><i class="fa fa-check"></i><b>2</b> Basic Knowledge</a>
<ul>
<li class="chapter" data-level="2.1" data-path="basic-knowledge.html"><a href="basic-knowledge.html#solution"><i class="fa fa-check"></i><b>2.1</b> Solution</a></li>
<li class="chapter" data-level="2.2" data-path="basic-knowledge.html"><a href="basic-knowledge.html#solubility"><i class="fa fa-check"></i><b>2.2</b> Solubility</a></li>
<li class="chapter" data-level="2.3" data-path="basic-knowledge.html"><a href="basic-knowledge.html#temperature"><i class="fa fa-check"></i><b>2.3</b> Temperature</a></li>
<li class="chapter" data-level="2.4" data-path="basic-knowledge.html"><a href="basic-knowledge.html#ph-and-poh---pondus-hydrogenium-and-pondus-oh"><i class="fa fa-check"></i><b>2.4</b> pH and pOH - pondus Hydrogenium and pondus OH</a></li>
<li class="chapter" data-level="2.5" data-path="basic-knowledge.html"><a href="basic-knowledge.html#mvph---millivolt-ph"><i class="fa fa-check"></i><b>2.5</b> MVpH - MilliVolt pH</a></li>
<li class="chapter" data-level="2.6" data-path="basic-knowledge.html"><a href="basic-knowledge.html#orp---oxidation-reduction-potential"><i class="fa fa-check"></i><b>2.6</b> ORP - Oxidation Reduction Potential</a></li>
<li class="chapter" data-level="2.7" data-path="basic-knowledge.html"><a href="basic-knowledge.html#do---dissolved-oxygen"><i class="fa fa-check"></i><b>2.7</b> DO - Dissolved Oxygen</a></li>
<li class="chapter" data-level="2.8" data-path="basic-knowledge.html"><a href="basic-knowledge.html#conductibility-spc"><i class="fa fa-check"></i><b>2.8</b> Conductibility (spc)</a></li>
<li class="chapter" data-level="2.9" data-path="basic-knowledge.html"><a href="basic-knowledge.html#tds---total-dissolved-solids"><i class="fa fa-check"></i><b>2.9</b> TDS - Total Dissolved Solids</a></li>
<li class="chapter" data-level="2.10" data-path="basic-knowledge.html"><a href="basic-knowledge.html#salinity"><i class="fa fa-check"></i><b>2.10</b> Salinity</a></li>
<li class="chapter" data-level="2.11" data-path="basic-knowledge.html"><a href="basic-knowledge.html#density"><i class="fa fa-check"></i><b>2.11</b> Density</a></li>
<li class="chapter" data-level="2.12" data-path="basic-knowledge.html"><a href="basic-knowledge.html#alcalinity"><i class="fa fa-check"></i><b>2.12</b> Alcalinity</a></li>
<li class="chapter" data-level="2.13" data-path="basic-knowledge.html"><a href="basic-knowledge.html#melting-temperature"><i class="fa fa-check"></i><b>2.13</b> Melting Temperature</a></li>
<li class="chapter" data-level="2.14" data-path="basic-knowledge.html"><a href="basic-knowledge.html#gas-laws"><i class="fa fa-check"></i><b>2.14</b> Gas laws</a></li>
<li class="chapter" data-level="2.15" data-path="basic-knowledge.html"><a href="basic-knowledge.html#plate-tectonics"><i class="fa fa-check"></i><b>2.15</b> Plate Tectonics</a></li>
<li class="chapter" data-level="2.16" data-path="basic-knowledge.html"><a href="basic-knowledge.html#subduction-zones"><i class="fa fa-check"></i><b>2.16</b> Subduction zones</a></li>
<li class="chapter" data-level="2.17" data-path="basic-knowledge.html"><a href="basic-knowledge.html#microbial-growth"><i class="fa fa-check"></i><b>2.17</b> Microbial growth</a></li>
<li class="chapter" data-level="2.18" data-path="basic-knowledge.html"><a href="basic-knowledge.html#binary-fission"><i class="fa fa-check"></i><b>2.18</b> Binary fission</a></li>
<li class="chapter" data-level="2.19" data-path="basic-knowledge.html"><a href="basic-knowledge.html#biofilm"><i class="fa fa-check"></i><b>2.19</b> Biofilm</a></li>
<li class="chapter" data-level="2.20" data-path="basic-knowledge.html"><a href="basic-knowledge.html#absorbance"><i class="fa fa-check"></i><b>2.20</b> Absorbance</a></li>
<li class="chapter" data-level="2.21" data-path="basic-knowledge.html"><a href="basic-knowledge.html#calibration-curve"><i class="fa fa-check"></i><b>2.21</b> Calibration Curve</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="geochemical-analyses.html"><a href="geochemical-analyses.html"><i class="fa fa-check"></i><b>3</b> Geochemical Analyses</a>
<ul>
<li class="chapter" data-level="3.1" data-path="geochemical-analyses.html"><a href="geochemical-analyses.html#sulphide"><i class="fa fa-check"></i><b>3.1</b> Sulphide</a></li>
<li class="chapter" data-level="3.2" data-path="geochemical-analyses.html"><a href="geochemical-analyses.html#iron"><i class="fa fa-check"></i><b>3.2</b> Iron</a></li>
<li class="chapter" data-level="3.3" data-path="geochemical-analyses.html"><a href="geochemical-analyses.html#doc---dissolved-organic-carbon"><i class="fa fa-check"></i><b>3.3</b> DOC - Dissolved Organic Carbon</a></li>
<li class="chapter" data-level="3.4" data-path="geochemical-analyses.html"><a href="geochemical-analyses.html#dic---dissolved-inorganic-carbon"><i class="fa fa-check"></i><b>3.4</b> DIC - Dissolved Inorganic Carbon</a></li>
<li class="chapter" data-level="3.5" data-path="geochemical-analyses.html"><a href="geochemical-analyses.html#ea---elemental-analysis"><i class="fa fa-check"></i><b>3.5</b> EA - Elemental Analysis</a></li>
<li class="chapter" data-level="3.6" data-path="geochemical-analyses.html"><a href="geochemical-analyses.html#icp-ms---inductively-coupled-plasma---mass-spectrometry"><i class="fa fa-check"></i><b>3.6</b> ICP-ms - Inductively Coupled Plasma - mass spectrometry</a></li>
<li class="chapter" data-level="3.7" data-path="geochemical-analyses.html"><a href="geochemical-analyses.html#ic---ion-chromatography"><i class="fa fa-check"></i><b>3.7</b> IC - Ion Chromatography</a></li>
<li class="chapter" data-level="3.8" data-path="geochemical-analyses.html"><a href="geochemical-analyses.html#isotope"><i class="fa fa-check"></i><b>3.8</b> Isotope</a></li>
<li class="chapter" data-level="3.9" data-path="geochemical-analyses.html"><a href="geochemical-analyses.html#pom---particulate-organic-matter"><i class="fa fa-check"></i><b>3.9</b> POM - Particulate Organic Matter</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="microbial-metabolisms.html"><a href="microbial-metabolisms.html"><i class="fa fa-check"></i><b>4</b> Microbial Metabolisms</a>
<ul>
<li class="chapter" data-level="4.1" data-path="microbial-metabolisms.html"><a href="microbial-metabolisms.html#phototrophy"><i class="fa fa-check"></i><b>4.1</b> Phototrophy</a></li>
<li class="chapter" data-level="4.2" data-path="microbial-metabolisms.html"><a href="microbial-metabolisms.html#chemotrophy"><i class="fa fa-check"></i><b>4.2</b> Chemotrophy</a></li>
<li class="chapter" data-level="4.3" data-path="microbial-metabolisms.html"><a href="microbial-metabolisms.html#autotrophy"><i class="fa fa-check"></i><b>4.3</b> Autotrophy</a></li>
<li class="chapter" data-level="4.4" data-path="microbial-metabolisms.html"><a href="microbial-metabolisms.html#heterotrophy"><i class="fa fa-check"></i><b>4.4</b> Heterotrophy</a></li>
<li class="chapter" data-level="4.5" data-path="microbial-metabolisms.html"><a href="microbial-metabolisms.html#photoautotrophy"><i class="fa fa-check"></i><b>4.5</b> Photoautotrophy</a></li>
<li class="chapter" data-level="4.6" data-path="microbial-metabolisms.html"><a href="microbial-metabolisms.html#photoheterotrophy"><i class="fa fa-check"></i><b>4.6</b> Photoheterotrophy</a></li>
<li class="chapter" data-level="4.7" data-path="microbial-metabolisms.html"><a href="microbial-metabolisms.html#chemoorganotrophy"><i class="fa fa-check"></i><b>4.7</b> Chemoorganotrophy</a></li>
<li class="chapter" data-level="4.8" data-path="microbial-metabolisms.html"><a href="microbial-metabolisms.html#chemolithotrophy"><i class="fa fa-check"></i><b>4.8</b> Chemolithotrophy</a></li>
<li class="chapter" data-level="4.9" data-path="microbial-metabolisms.html"><a href="microbial-metabolisms.html#fermentation"><i class="fa fa-check"></i><b>4.9</b> Fermentation</a></li>
<li class="chapter" data-level="4.10" data-path="microbial-metabolisms.html"><a href="microbial-metabolisms.html#methanogenesis"><i class="fa fa-check"></i><b>4.10</b> Methanogenesis</a></li>
<li class="chapter" data-level="4.11" data-path="microbial-metabolisms.html"><a href="microbial-metabolisms.html#sulphate-reduction"><i class="fa fa-check"></i><b>4.11</b> Sulphate-reduction</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="molecular-biology.html"><a href="molecular-biology.html"><i class="fa fa-check"></i><b>5</b> Molecular Biology</a>
<ul>
<li class="chapter" data-level="5.1" data-path="molecular-biology.html"><a href="molecular-biology.html#s-rrna"><i class="fa fa-check"></i><b>5.1</b> 16S rRNA</a></li>
<li class="chapter" data-level="5.2" data-path="molecular-biology.html"><a href="molecular-biology.html#shotgun-metagenomic-sequencing"><i class="fa fa-check"></i><b>5.2</b> Shotgun metagenomic sequencing</a></li>
<li class="chapter" data-level="5.3" data-path="molecular-biology.html"><a href="molecular-biology.html#electrophoresis-gel-dna"><i class="fa fa-check"></i><b>5.3</b> Electrophoresis gel DNA</a></li>
<li class="chapter" data-level="5.4" data-path="molecular-biology.html"><a href="molecular-biology.html#pcr"><i class="fa fa-check"></i><b>5.4</b> PCR</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="biogeochemical-cycles.html"><a href="biogeochemical-cycles.html"><i class="fa fa-check"></i><b>6</b> Biogeochemical Cycles</a>
<ul>
<li class="chapter" data-level="6.1" data-path="biogeochemical-cycles.html"><a href="biogeochemical-cycles.html#water-cycle"><i class="fa fa-check"></i><b>6.1</b> Water cycle</a></li>
<li class="chapter" data-level="6.2" data-path="biogeochemical-cycles.html"><a href="biogeochemical-cycles.html#carbon-cycle"><i class="fa fa-check"></i><b>6.2</b> Carbon cycle</a></li>
<li class="chapter" data-level="6.3" data-path="biogeochemical-cycles.html"><a href="biogeochemical-cycles.html#nitrogen-cycle"><i class="fa fa-check"></i><b>6.3</b> Nitrogen cycle</a></li>
<li class="chapter" data-level="6.4" data-path="biogeochemical-cycles.html"><a href="biogeochemical-cycles.html#phosphorus-cycle"><i class="fa fa-check"></i><b>6.4</b> Phosphorus cycle</a></li>
<li class="chapter" data-level="6.5" data-path="biogeochemical-cycles.html"><a href="biogeochemical-cycles.html#sulphur-cycle"><i class="fa fa-check"></i><b>6.5</b> Sulphur cycle</a></li>
<li class="chapter" data-level="6.6" data-path="biogeochemical-cycles.html"><a href="biogeochemical-cycles.html#hydrogen-cycle"><i class="fa fa-check"></i><b>6.6</b> Hydrogen cycle</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="extreme-environments.html"><a href="extreme-environments.html"><i class="fa fa-check"></i><b>7</b> Extreme Environments</a>
<ul>
<li class="chapter" data-level="7.1" data-path="extreme-environments.html"><a href="extreme-environments.html#sea-ice-permafrost-and-polar-regions"><i class="fa fa-check"></i><b>7.1</b> Sea ice, permafrost and polar regions</a></li>
<li class="chapter" data-level="7.2" data-path="extreme-environments.html"><a href="extreme-environments.html#cold-seeps-and-mud-volcanoes"><i class="fa fa-check"></i><b>7.2</b> Cold seeps and mud volcanoes</a></li>
<li class="chapter" data-level="7.3" data-path="extreme-environments.html"><a href="extreme-environments.html#shallow-water-hydrothermal-vents"><i class="fa fa-check"></i><b>7.3</b> Shallow water-hydrothermal vents</a></li>
<li class="chapter" data-level="7.4" data-path="extreme-environments.html"><a href="extreme-environments.html#hot-springs-fumaroles-and-mud-volcanoes"><i class="fa fa-check"></i><b>7.4</b> Hot-springs, fumaroles and mud volcanoes</a></li>
<li class="chapter" data-level="7.5" data-path="extreme-environments.html"><a href="extreme-environments.html#hyperacidic-lakes-and-volcanoes"><i class="fa fa-check"></i><b>7.5</b> Hyperacidic lakes and volcanoes</a></li>
<li class="chapter" data-level="7.6" data-path="extreme-environments.html"><a href="extreme-environments.html#deserts-and-arid-environments"><i class="fa fa-check"></i><b>7.6</b> Deserts and arid environments</a></li>
<li class="chapter" data-level="7.7" data-path="extreme-environments.html"><a href="extreme-environments.html#acid-mine-drainage"><i class="fa fa-check"></i><b>7.7</b> Acid mine drainage</a></li>
<li class="chapter" data-level="7.8" data-path="extreme-environments.html"><a href="extreme-environments.html#deep-sea-anoxic-lakes-and-brines"><i class="fa fa-check"></i><b>7.8</b> Deep-sea anoxic lakes and brines</a></li>
<li class="chapter" data-level="7.9" data-path="extreme-environments.html"><a href="extreme-environments.html#deep-sea-hydrothermal-vents"><i class="fa fa-check"></i><b>7.9</b> Deep-sea hydrothermal vents</a></li>
<li class="chapter" data-level="7.10" data-path="extreme-environments.html"><a href="extreme-environments.html#serpentinizing-environments"><i class="fa fa-check"></i><b>7.10</b> Serpentinizing environments</a></li>
<li class="chapter" data-level="7.11" data-path="extreme-environments.html"><a href="extreme-environments.html#deep-sea-sediments-and-trenches"><i class="fa fa-check"></i><b>7.11</b> Deep-sea sediments and trenches</a></li>
<li class="chapter" data-level="7.12" data-path="extreme-environments.html"><a href="extreme-environments.html#soda-lakes-and-hypersaline-lakes"><i class="fa fa-check"></i><b>7.12</b> Soda lakes and hypersaline lakes</a></li>
<li class="chapter" data-level="7.13" data-path="extreme-environments.html"><a href="extreme-environments.html#nuclear-contaminated-sites"><i class="fa fa-check"></i><b>7.13</b> Nuclear contaminated sites</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="polygons-plot.html"><a href="polygons-plot.html"><i class="fa fa-check"></i><b>8</b> Polygons Plot</a>
<ul>
<li class="chapter" data-level="8.0.1" data-path="polygons-plot.html"><a href="polygons-plot.html#polygon-plot-features"><i class="fa fa-check"></i><b>8.0.1</b> Polygon plot features</a></li>
<li class="chapter" data-level="8.0.2" data-path="polygons-plot.html"><a href="polygons-plot.html#the-inception"><i class="fa fa-check"></i><b>8.0.2</b> The inception</a></li>
<li class="chapter" data-level="8.0.3" data-path="polygons-plot.html"><a href="polygons-plot.html#reading-the-polygon-plot"><i class="fa fa-check"></i><b>8.0.3</b> Reading the Polygon plot</a></li>
<li class="chapter" data-level="8.0.4" data-path="polygons-plot.html"><a href="polygons-plot.html#using-the-polygon-plots-the-shiny-app"><i class="fa fa-check"></i><b>8.0.4</b> Using the Polygon plots: the Shiny app</a></li>
<li class="chapter" data-level="8.0.5" data-path="polygons-plot.html"><a href="polygons-plot.html#polygons-plot-array"><i class="fa fa-check"></i><b>8.0.5</b> Polygons plot array</a></li>
<li class="chapter" data-level="8.0.6" data-path="polygons-plot.html"><a href="polygons-plot.html#list-of-future-features"><i class="fa fa-check"></i><b>8.0.6</b> List of future features</a></li>
<li class="chapter" data-level="8.0.7" data-path="polygons-plot.html"><a href="polygons-plot.html#how-to-cite-the-polygon-plot"><i class="fa fa-check"></i><b>8.0.7</b> How to cite the Polygon plot</a></li>
<li class="chapter" data-level="8.0.8" data-path="polygons-plot.html"><a href="polygons-plot.html#questions-support-and-contribute"><i class="fa fa-check"></i><b>8.0.8</b> Questions, support and contribute!</a></li>
<li class="chapter" data-level="8.0.9" data-path="polygons-plot.html"><a href="polygons-plot.html#gallery-of-polygons"><i class="fa fa-check"></i><b>8.0.9</b> Gallery of Polygons</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="contacts.html"><a href="contacts.html"><i class="fa fa-check"></i><b>9</b> Contacts</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">G.Lab. Book</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="molecular-biology" class="section level1 hasAnchor" number="5">
<h1><span class="header-section-number">Chapter 5</span> Molecular Biology<a href="molecular-biology.html#molecular-biology" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="s-rrna" class="section level2 hasAnchor" number="5.1">
<h2><span class="header-section-number">5.1</span> 16S rRNA<a href="molecular-biology.html#s-rrna" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>This test analyzes a bacterial gene called 16S rRNA – a sort of identity card for microbes – and provides the name of all the bacteria present in the sample and their abundance as a result.<br />
It is the most studied gene for microbial phylogeny; it is one of the components of the lower subunit of the bacterial ribosome.<br />
The gene, that codes for 16S RNA (component of the small subunit of the bacterial ribosome), is approximately 1,500 bp long. A 2-3% change in nucleotide sequence (30-45 bp) is considered indicative of a different species.<br />
The approach of building phylogenetic trees based on 16S sequence alignment is the most widespread and was used for the first time in 1977 by Carl Woese to distinguish the domain of Archaea from that of Bacteria.<br />
</p>
</div>
<div id="shotgun-metagenomic-sequencing" class="section level2 hasAnchor" number="5.2">
<h2><span class="header-section-number">5.2</span> Shotgun metagenomic sequencing<a href="molecular-biology.html#shotgun-metagenomic-sequencing" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>High-throughput sequencing approaches enable genomic analyses ideally of all microbes in a sample, not just those that are amenable to cultivation.<br />
Shotgun metagenomic sequencing allows researchers to comprehensively sample all genes from all organisms in a given complex sample. The method allows microbiologists to evaluate bacterial and metabolic diversity and detect the abundance of microbes in various environments.<br />
A typical shotgun metagenomics study comprises five steps, after the initial study design:</p>
<ul>
<li>the collection, processing and sequencing of the samples;<br />
</li>
<li>preprocessing of the sequencing reads;<br />
</li>
<li>sequence analysis to profile taxonomic, functional and genomic features of the microbiome;<br />
</li>
<li>statistical and biological post-processing analysis;<br />
</li>
<li>validation.<br />
</li>
</ul>
</div>
<div id="electrophoresis-gel-dna" class="section level2 hasAnchor" number="5.3">
<h2><span class="header-section-number">5.3</span> Electrophoresis gel DNA<a href="molecular-biology.html#electrophoresis-gel-dna" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Nucleic acid gel electrophoresis is a molecular biology technique that allows for the separation, identification, and purification of DNA and RNA fragments based on size and charge. In this technique, DNA and RNA molecules are separated by applying an electric field that moves the negatively charged nucleic acids through a matrix of:</p>
<ul>
<li><strong>Agarose</strong>, which is extracted from seaweed, is appropriate for separating DNA fragments ranging in size from a few hundred base pairs to about 20 kb.<br />
</li>
<li><strong>Polyacrylamide</strong> is preferred for proteins and smaller DNA fragments.<br />
In general, the gel matrix acts like a sieve, allowing shorter molecules to move more quickly through the pores of the gel than longer molecules.<br />
</li>
</ul>
<p>The equipment and supplies for conducting agarose gel electrophoresis in DNA studies include electrophoresis chamber and power supply, gel casting trays, sample combs, electrophoresis buffer, loading buffer, ethidium bromide (EtBr), and transilluminator. 
One method of staining DNA is to expose it to the dye EtBr. EtBr intercalates between the stacked bases of nucleic acids and fluoresces red–orange when illuminated with ultraviolet (UV) light. EtBr should be handled with care for its toxic and mutagenic nature; this is the reason why, it is substituted by other dyes like Sybr Green.<br />
Agarose gels are cast by melting the agarose powder dissolved in a specific buffer until a clear, transparent solution is achieved. The melted gel is then poured on a tray and allowed to solidify. Upon solidification, the DNA samples are loaded into the sample wells and gel is run at a voltage and for a time period that will perform optimal separation. One well is reserved for a DNA ladder, a standard reference that contains DNA fragments of known lengths. When an electric field is applied across the gel, DNA migrates toward the anode.<br />
A well-defined “line” of DNA on a gel is called a band. Each band contains a large number of DNA fragments of the same size that have all traveled as a group to the same location. By comparing the bands of a sample to the DNA ladder, we can determine its approximate size.<br />
For linear fragments of DNA and/or RNA the migration distance is inversely proportional to the size of the molecule (i.e. its length in bases).<br />
Single-stranded nucleic acids (like RNA) tend to form complex secondary structures, so their “electrophoretic pathway” is strongly influenced by how they fold. Before being separated by electrophoresis, RNA must be denatured to keep the molecules in linear shape. Commonly used denaturing agents: heat, formaldehyde, formamide, urea.<br />
</p>
</div>
<div id="pcr" class="section level2 hasAnchor" number="5.4">
<h2><span class="header-section-number">5.4</span> PCR<a href="molecular-biology.html#pcr" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Polymerase chain reaction (PCR) is a technique for exponential ampliﬁcation of a fragment of DNA, using a heat-stable DNA polymerase and an automated heat block that is capable of rapid changes of temperature, inducing the enzymatic synthesis of millions of copies of a specific DNA segment.<br />
It can be schematized in different steps:</p>
<ul>
<li><strong>First stage:</strong> The template DNA molecule is initially denatured to two single strands by heating to high temperature (typically 90–95 °C). Two small oligonucleotides that are complementary to sequences on opposite strands of the template molecule are used as primers, allowing the binding for the DNA polymerase.<br />
</li>
<li><strong>Second stage:</strong> the single-stranded oligonucleotide primers anneal to the denatured template molecule, during a cooling phase.<br />
</li>
<li><strong>Third stage:</strong> the new strand extends from the annealed primer in a 5’-3’ direction by the heat-stable polymerase. This is performed at the optimum temperature for the polymerase (68–72 °C).<br />
</li>
</ul>
<p>The most commonly used polymerase is the enzyme isolated from Thermus aquaticus (Taq DNA polymerase).<br />
After the ﬁrst cycle, each template molecule has been ampliﬁed to two molecules. These in turn are denatured in the next cycle and ampliﬁed to produce four molecules. The four molecules are ampliﬁed to eight in the third cycle, and so on. Each successive cycle effectively doubles the amount of DNA product. The three-stage cycle of denaturation, annealing, and primer extension is repeated 25–40 times in a typical PCR procedure.<br />
PCR products are visualized by electrophoresis gel technique followed by staining with ﬂuorescent dyes or by hybridization to labeled oligonucleotide probes.<br />
Species specific primers are used. The most used sequence is rRNA that includes the gene for 16S rRNA.<br />
The reagents that are used are:</p>
<ul>
<li><strong>Buffer Tris-HCl:</strong> keeps the pH stable in order to generate an environment suitable for the reaction.<br />
</li>
<li><strong>Primers:</strong> single-stranded oligonucleotides of approximately 18-25 bases that allow the reaction to be started. The primers are complementary, one at the 3’ end and the other at the 5’ end of the DNA segment you want to amplify. In particular, the forward primer will bind to the 3’-5’ antiparallel strand, delimiting the beginning of the synthesis, while the reverse primer will bind to the other strand, delimiting the end of the synthesis.<br />
</li>
<li><strong><span class="math inline">\(MgCl_{2}\)</span>:</strong> Cofactor of Taq DNA polymerase that influences primer hybridization.<br />
</li>
<li><strong>Deoxynucleoside triphosphates (dNTPs-dATP, dTTP, dGTP, dCTP):</strong> they are necessary for the synthesis of new strands.<br />
</li>
<li><strong>Taq DNA polymerase:</strong> heat-stable polymerase.<br />
</li>
<li><strong>DNA sample</strong> of interest.<br />
</li>
</ul>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="microbial-metabolisms.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="biogeochemical-cycles.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": false
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["GlabBook.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
